### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by visortelle**

### CODE REVIEW

File: go/core.rst [hunk 1]
```diff
@@ -527,7 +527,7 @@ Attributes
 | :param:`msan`              | :type:`string`              | :value:`auto`                         |
 +----------------------------+-----------------------------+---------------------------------------+
 | This is one of the `mode attributes`_ that controls whether to instrument                        |
-| code for memory santization. It may be :value:`on`, :value:`off`, or                             |
+| code for memory sanitization. It may be :value:`on`, :value:`off`, or                             |
```
Comment (jayconrod) [inline go/core.rst:530]: Please make sure the right edge is aligned. The table won't render without this.

File: go/core.rst [hunk 2]
```diff
@@ -743,7 +743,7 @@ Attributes
...
```
Comment (jayconrod) [inline go/core.rst:746]: This one, too.

File: go/core.rst (hunk 1 above)
Comment (visortelle) [reply go/core.rst]: Fixed

File: go/core.rst (hunk 2 above)
Comment (visortelle) [reply go/core.rst]: Fixed

**Pull request merged**